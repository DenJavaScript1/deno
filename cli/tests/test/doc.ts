/**
 * ```js
 * import { ok } from "./doc.ts";
 *
 * ok();
 * ```
 *
 * ```ts
 * import { ok } from "./doc.ts";
 *
 * ok();
 * ```
 *
 */
export function ok() {
  // no-op
}

/**
 * ```js
 * import { fail } from "./doc.ts";
 *
 * fail();
 * ```
 *
 * ```ts
 * import { fail } from "./doc.ts";
 *
 * fail();
 * ```
 */
export function fail() {
  throw new Error();
}

/**
 * ```js
 * # import { hidden } from "./doc.ts";
 *
 * hidden();
 * ```
 *
 * ```ts
 * # import { hidden } from "./doc.ts";
 *
 * hidden();
 * ```
 */
export function hidden() {
  // no-op
}

