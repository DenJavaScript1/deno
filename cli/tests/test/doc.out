Check [WILDCARD]/doc.ts
Check [WILDCARD]/doc.ts:20-25
Check [WILDCARD]/doc.ts:26-31
Check [WILDCARD]/doc.ts:2-7
Check [WILDCARD]/doc.ts:8-13
running 0 tests from [WILDCARD]/doc.ts
running 4 tests from [WILDCARD]/doc
test [WILDCARD]/doc.ts:20-25 ... FAILED [WILDCARD]
test [WILDCARD]/doc.ts:26-31 ... FAILED [WILDCARD]
test [WILDCARD]/doc.ts:2-7 ... ok [WILDCARD]
test [WILDCARD]/doc.ts:8-13 ... ok [WILDCARD]

failures:

[WILDCARD]/doc.ts:20-25
Error
    at fail ([WILDCARD]/doc.ts:33:9)
    at [WILDCARD]/doc.ts:20-25.ts:3:1

[WILDCARD]/doc.ts:26-31
Error
    at fail ([WILDCARD]/doc.ts:33:9)
    at [WILDCARD]/doc.ts:26-31.ts:3:1

failures:

	[WILDCARD]/doc.ts:20-25
	[WILDCARD]/doc.ts:26-31

test result: FAILED. 2 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out [WILDCARD]
